# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1514843916 21600
#      Mon Jan 01 15:58:36 2018 -0600
# Node ID 9d0522881498929857d5abdfb29b534246be1c7a
# Parent  9c178e5252e430e4313e9e45c2f3825a19cf6f31
doc: Make code-example apostrophe in the manual cut-and-paste friendly (bug #52775).

* macros.texi: Set Texinfo '@codequoteundirected on' flag.

diff --git a/doc/interpreter/macros.texi b/doc/interpreter/macros.texi
--- a/doc/interpreter/macros.texi
+++ b/doc/interpreter/macros.texi
@@ -102,8 +102,11 @@ See also: \args\.
     \global\def\urlcolor{\rgbDarkRed}
   \fi
   \ifx\urefurlonlylinktrue\thisisundefined
     \relax
   \else
     \global\urefurlonlylinktrue
   \fi
 @end tex
+
+@c Make the apostrophe in code examples cut-and-paste friendly.
+@codequoteundirected on
